.eslintrc.js

```diff
@@ -55,7 +55,8 @@ module.exports = {
         }],
         'no-return-assign': ['error', 'always'],
         'no-param-reassign': 'error',
-
+        'no-unused-vars': 'off',
+        '@typescript-eslint/no-unused-vars': ['error'],
```

Comment:
fix a typescript error where function arguments with a function type (e.g. when passing $t to a function) shows an unused variable error